server/interceptor/interceptor.go

```diff
@@ -405,8 +405,21 @@ func newRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
 			info.tokens = len(req.GetLeaves())
 		case *trillian.AddSequencedLeavesRequest:
 			info.tokens = len(req.GetLeaves())
+		case *trillian.GetLeavesByRangeRequest:
+			info.tokens = 1
+			if c := req.GetCount(); c > 1 {
+				info.tokens = int(c)
+			}
+		case *trillian.GetLeavesByIndexRequest:
+			info.tokens = len(req.GetLeafIndex())
+		case *trillian.GetLeavesByHashRequest:
+			info.tokens = len(req.GetLeafHash())
 		case *trillian.SetMapLeavesRequest:
 			info.tokens = len(req.GetLeaves())
+		case *trillian.GetMapLeavesRequest:
+			info.tokens = len(req.GetIndex())
+		case *trillian.GetMapLeavesByRevisionRequest:
+			info.tokens = len(req.GetIndex())
```

Comment:
I assume we're happy not requiring any quota for requests that don't actually ask for any leaves? I guess they'll be extremely cheap, so that seems fine.

Reply:
Also, I think requests that use sequencing-based quotas definitely need to have 0 tokens for 0 queued leaves, to make the sums add up.